web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -222,9 +234,42 @@ export const ConnectMyComputerContextProvider: FC<{
     }
   }
 
+  const agentIsNotStarted =
+    currentAction.kind === 'observe-process' &&
```

Comment:
Hmm, should `observe-process` be our default? What we had a special action kind like `none` to represent this use case? Literally just `{kind : 'none' }`.

Reply:
At first I thought that's a great idea, but while testing it, I noticed that a bug. When I pressed Command+R, the UI showed  'Agent no running' while the agent in the main process was still working. Defaulting to `observe-process` allows to alway show the current agent state, even if the agent started before the UI was shown. I know that it is generally possible only in dev mode, but I think it is better to stay with `observe-process`.

Reply:
Hmm I think you're right. I'd consider dev-only issues as sort of secondary problems. But I was mostly worried about us not being able to differentiate between "We've never even touched the agent in this session" vs "The user has stopped the agent".

But since this code is run only once, then I suppose it's okay.

Reply:
Ah, then we're good.

If we wanted to be really precise, then a separate action kind here would be useful anyway, since it clearly delineates that no action was taken vs reusing a single action for two different purposes.

But as you said, then we'd have to take care of the issue during development etc., so we can keep it as is.